core/tests/test_apps.common.cbor.py

```diff
@@ -11,9 +11,43 @@
     create_embedded_cbor_bytes_header,
     decode,
     encode,
-    encode_chunked,
     encode_streamed,
 )
+from trezor import utils
+
+# NOTE: moved into tests not to occupy flash space
+# in firmware binary, when it is not used in production
+def encode_chunked(value: "Value", max_chunk_size: int) -> "Iterator[bytes]":
```

Comment:
instead, drop this code as well as `test_encode_chunked`

Reply:
Dropped in [1a413bb](https://github.com/trezor/trezor-firmware/pull/2525/commits/1a413bbffa528c9e781d76ee0efa91e336add245)